airflow/utils/json.py

```diff
@@ -59,7 +59,11 @@ def default(self, o: Any) -> Any:
             data = serialize(o)
             if isinstance(data, dict) and DATA in data:
                 return data[DATA]
-
+        if isinstance(o, bytes):
+            try:
+                return o.decode()
+            except Exception:
+                return o.__repr__()
```

Comment:
```suggestion
                return repr(o)
```

Same for the one below